src/shared/copy/notifications/categories/accounts-users-orgs.ts

```diff
@@ -61,6 +71,29 @@ export const orgRenameFailed = (orgName): Notification => ({
   message: `Failed to update organization "${orgName}"`,
 })
 
+export const updateIdentityFailed = (): Notification => ({
+  ...defaultErrorNotification,
+  message: 'Error retrieving user identity. Please refresh this page.',
+})
+
+export const updateBillingFailed = (): Notification => ({
+  ...defaultErrorNotification,
+  message:
+    'Error retrieving account billing provider. Please refresh this page.',
+})
+
+export const updateOrgFailed = (): Notification => ({
+  ...defaultErrorNotification,
+  message:
+    'Error retrieving new organization information. Please refresh this page.',
+})
+
+export const updateQuartzOrganizationsFailed = (): Notification => ({
+  ...defaultErrorNotification,
+  message:
+    'We were unable to retrieve the list of your InfluxData organizations. Please refresh this page.',
+})
+
```

Comment:
Absent any other organization scheme, I'd like us to default to alphabetization, since that will probably be the easiest way to find functions in a file like this.

Reply:
SGTM. 

There was a comment in there, `/* USERS NOTIFICATIONS */` that seems to separate out notifications relating to user invites. But there's not really a logical reason why those should be distinct in the file, so I'll impose alpha order here.